test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -24,32 +34,17 @@ test.describe(
 
 		test( 'default to a list of pages if there are no menus', async ( {
 			editor,
-			page,
 		} ) => {
-			const pagesRequest = page
-				.waitForRequest( ( response ) => {
-					return response.url().includes( 'rest_route=/wp/v2/pages' );
-				} )
-				.catch( () => {} );
-
-			const menusRequest = page
-				.waitForRequest( ( response ) => {
-					return response.url().includes( 'rest_route=/wp/v2/menus' );
-				} )
-				.catch( () => {} );
-
 			await editor.insertBlock( { name: 'core/navigation' } );
 
-			await pagesRequest;
-			await menusRequest;
```

Comment:
We don't need to explicitly wait for these implementation details. Instead, the recommended way is to wait for what the users expect. In this case, we can just wait for the page list block to be visible.

Reply:
That would be 100% ideal. Let's hope we can do this. But isn't it potentially flaky to rely on network resolving faster than the Locator timeout value? Should I bump the timeout?